ui/src/Components/notification/index.jsx

```diff
@@ -0,0 +1,86 @@
+import { useState } from "react";
+
+/**
+ * a pubsub kinda implementation, basically it will save a function in closure with varaible called "fn"
+ * when i use a function called subcribe and pass it with the function i want to save
+ * then when i will call publish function with "data" parameter it will call the function present in "fn"
+ * with the "data" as a arguement
+ */
+export const NotificationQue = () => {
+  let fn = () => {};
+  const subscribe = (callback) => {
+    fn = callback;
+  };
+  const publish = (data) => {
+    return fn(data);
+  };
+  return { subscribe, publish };
+};
+
+/**
+ * now this NotificationHandler can be imported in any component and can be called to create a
+ * notification
+ */
+export const { subscribe, publish: NotificationHandler } = NotificationQue();
+
+/**
+ * when we have decided what kind of resources we want to use for icons there also be an icon rendered according to
+ * type of the notification
+ *
+ *
+ * @param {{message : string, type : string}} data the structure of notification that has to be passed down to the NotificationHandler
+ */
+export const NotificationElement = ({ data }) => {
+  const typeToColor = {
+    success: "rgb(95, 214, 95)",
+    warning: "rgb(238, 241, 41)",
+    error: "rgb(253, 81, 81)",
```

Comment:
Would it be better to create a constant for the success, warning, and error so that
1. it could be refactored through Source of Point in future
2. Having one place for the color observation changes

Reply:
Constants here are component specific. So, Success, Warning, and Error are related to the Notification componeny. Breaking them into separate constant would break when some other component will use and try to change the value

Reply:
Shouldn't we be maintaining a pallette of colours....
Lets say we change the color of red by 10% 
Shouldn't this be ensured every component about the changes?
We might utilise banners as well in future, wouldn't it be good to have global colors?

Let me know if I am overthinking or thinking in different directions

Reply:
great and Valid point. We are using tailwind for the global CSS. So, your suggestion now makes me think about why not we leverage those classes. 

cc @ivinayakg